### DISCUSSION TITLE
Add support of dynamic log level

### DISCUSSION THREAD

**Discussion by zuston** [fast/logforth#94] [Q&A] [answered]
Thanks for your great work of logforth, I used this into the storage service.

if logforth is used in the production environment, sometimes we want to temporarily adjust the log level to print out some logs for digging. Is this possible to support this in logforth?

**Comment by tisonkun** [comment, marked as answer]
> temporarily adjust the log level to print out some logs for digging

For this purpose, in our use case, we typically use the `RUST_LOG` env to enable debug level logs, like:

``` shell
RUST_LOG="debug" ./your/app
# or 
RUST_LOG="info,specific_mod=debug" ./your/app
```

This follows the most common used [env_filter](https://docs.rs/logforth/latest/logforth/filter/env_filter/struct.EnvFilter.html)

If you'd like to use more dynamic rules, you can use the [CustomFilter](https://docs.rs/logforth/latest/logforth/filter/struct.CustomFilter.html) for define your rules, and even with a global state to check so that you can modify it at runtime.

I'm glad to hear your concrete use case and see if we can integrate it at the upstream.
